js/controller.js

```diff
@@ -668,6 +697,11 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         width: ""
       });
       this.state.forceControlBarVisible = (this.state.pluginsElement.children().length > 0);
+
+      if (this.state.midVideo) {
+        this.mb.publish(OO.EVENTS.PAUSE);
```

Comment:
Until we enable the flash, the call to pause the main content doesn't happen, so we have the flash alert visible and the main video playing in the background unless we pause it.